python/kiss_icp/tools/visualizer.py

```diff
@@ -20,223 +20,301 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-import copy
 import importlib
 import os
 from abc import ABC
-from functools import partial
-from typing import Callable, List
 
 import numpy as np
 
-CYAN = np.array([0.24, 0.898, 1])
-RED = np.array([128, 0, 0]) / 255.0
-BLACK = np.array([0, 0, 0]) / 255.0
-BLUE = np.array([0.4, 0.5, 0.9])
-GRAY = np.array([0.4, 0.4, 0.4])
-SPHERE_SIZE = 0.20
+from kiss_icp.config.parser import KISSConfig
+
+BACKGROUND_COLOR = [0.8470, 0.8588, 0.8863]
+FRAME_COLOR = [0.2156, 0.2471, 0.3176]
+KEYPOINTS_COLOR = [0.8588, 0.3294, 0.3803]
+LOCAL_MAP_COLOR = [0.3451, 0.6431, 0.6902]
+TRAJECTORY_COLOR = [0.1058, 0.1059, 0.1176]
+FRAME_PTS_SIZE = 0.2
+KEYPOINTS_PTS_SIZE = 0.3
+MAP_PTS_SIZE = 0.1
 
 
 class StubVisualizer(ABC):
-    def __init__(self):
+    def __init__(self, config: KISSConfig):
         pass
 
-    def update(self, source, keypoints, target_map, pose):
+    def update(self, source, keypoints, target_map, pose, lest_time):
         pass
 
 
-class RegistrationVisualizer(StubVisualizer):
-    # Public Interaface ----------------------------------------------------------------------------
-    def __init__(self):
-        try:
-            self.o3d = importlib.import_module("open3d")
-        except ModuleNotFoundError as err:
-            print(f'open3d is not installed on your system, run "pip install open3d"')
-            exit(1)
+def start_pause_callback():
+    button_name = "PAUSE" if Kissualizer.play_mode else "START"
```

Comment:
Not sure if this is the render, but should we add
`START [Space]` and things like that, now the keyboard still works, which I love, as I'm not a mouse guy, but the letters for command are hidden.

The other option would be to spit in the user face the commands, like before

Reply:
I was thinking the same. Probably I can add a section in the GUI with the list of commands, or just add the key "on the buttons". I'll test both and post a screenshot.

Reply:
![Screenshot from 2024-07-31 16-59-20](https://github.com/user-attachments/assets/50741900-9978-4dd8-96d2-bd7c4c2039e2)
This is how it appears now, I kinda like it. Let me know what do you think

Reply:
fantastic